src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
Then you can ignore this pull request. I am still a very novice javascript programmer, but I try to help wherever I can. Widgets are now the biggest pain in the..... when moving from Mx5 to Mx6.